src/main/java/uk/gov/hmcts/reform/sscs/service/evidence/EvidenceUploadService.java

```diff
@@ -404,6 +405,7 @@ protected void buildScannedDocumentByGivenSscsDoc(SscsCaseData sscsCaseData, Ssc
                     emptyIfNull(audioVideoEvidence));
             sscsCaseData.setAudioVideoEvidence(newAudioVideoEvidenceList);
         }
+        setHasUnprocessedAudioVideoEvidenceFlag(sscsCaseData);
```

Comment:
Is there any existing unit tests we can use to check the value of the field set is asserted?